uvicorn/supervisors/statreload.py

```diff
@@ -30,9 +30,9 @@ def handle_fds(target, fd_stdin, **kwargs):
 
     def run(self, target, *args, **kwargs):
         pid = os.getpid()
-        logger = self.config.logger_instance
+        access_logger = self.config.access_logger_instance
 
-        logger.info("Started reloader process [{}]".format(pid))
+        access_logger.info("Started reloader process [{}]".format(pid))
```

Comment:
It'd be really useful to know which logger gunicorn uses for this kind of messaging.

Reply:
ok I dig a little
Gunicorn deals with reloading in [here](https://github.com/benoitc/gunicorn/blob/235f06c3aff23f72edd17f2054e29cb49102002e/gunicorn/workers/base.py#L120-L122):
```
        if self.cfg.reload:
            def changed(fname):
                self.log.info("Worker reloading: %s modified", fname)
```
Now self.log in this context is <gunicorn.glogging.Logger object at 0x7f137990bac8> which has in it's [dunder init ](https://github.com/benoitc/gunicorn/blob/235f06c3aff23f72edd17f2054e29cb49102002e/gunicorn/glogging.py#L185-L195) the 2 gunicorn loggers:
```
    def __init__(self, cfg):
        self.error_log = logging.getLogger("gunicorn.error")
        self.error_log.propagate = False
        self.access_log = logging.getLogger("gunicorn.access")
```

when it calls ` self.log.info("Worker reloading: %s modified", fname)` it spits it to the error_logger as you can see [here](https://github.com/benoitc/gunicorn/blob/235f06c3aff23f72edd17f2054e29cb49102002e/gunicorn/glogging.py#L261):

```
    def info(self, msg, *args, **kwargs):
        self.error_log.info(msg, *args, **kwargs)
```

More generally, it seems to me that everything goes to the `error_logger` in gunicorn except ONCE [here](https://github.com/benoitc/gunicorn/blob/235f06c3aff23f72edd17f2054e29cb49102002e/gunicorn/glogging.py#L327-L346)

Reply:
> It'd be really useful to know which logger gunicorn uses for this kind of messaging.

so the answer is clearly error_logger in case of gunicorn